##追踪部分位置环P
p_xy : 1.4

## 单位:米
R_outside : 1.2

## 单位:米
R_inside : 0.8

## 追踪部分速度限幅
vel_track_max : 0.3

## 大圈比例系数
p_R : 0.6

## 小圈比例系数
p_r : 1.2

## 避障部分最大速度
vel_collision_max : 0.30

## 总速度最大速度
vel_sp_max : 0.3


## 激光雷达范围设定
range_min : 0
range_max : 359

## 
lidar_max_distance : 6.0